src/toil/batchSystems/singleMachine.py

```diff
@@ -221,9 +221,16 @@ def daddy(self):
                             # cores/memory/disk individually)?
                             self.inputQueue.put(args)
                             break
+                        elif result is not False:
+                            #Result is a PID
+
+                            if self._outbox is not None:
+                                # Annotate the job with the PID generated.
+                                self._outbox.publish(
+                                   BatchSystemMessage(jobID, result, self.__class__.__name__))
 
                         # Otherwise it's a PID if it succeeded, or False if it couldn't
```

Comment:
I think now "otherwise" (to all the if cases above), the result can only be `False`.